docs/migration/5_0.md

```diff
@@ -40,3 +40,10 @@ under the section `headerDisplayOnly`.
 #### Translations (i18n) changes
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
+
+### PaymentFormComponent
+
+#### Translations (i18n) changes
+
+- Change contents of:
+  `payment.paymentForm.setAsDefault`,
```

Comment:
I would also add a change from what to what, for example - 

- Change contents of `payment.paymentForm.setAsDefault` from `Set as default` to `Set as default payment method` for screen reader improvements